fix: 로그인 버튼 클릭시 touch event cancel 현상 수정해요

Do-hyun-Kim opened the pull request:
## 🔵PR을 올리기 전 아래 사항을 확인해주세요.
- 구현한 로직과 기능이 올바르게 작동되는지 충분히 테스트해주세요.
- 코드의 성능이나 메모리 효율성이 적절하게 고려되었는지, 불필요한 코드가 없는지 검토해주세요.
- 이번 PR에서 구현된 주요 기능이나 해결된 문제에 대해 자세히 서술해주세요.
(위 내용은 지워주세요)



## 😽 BBButton Touch Cancel 현상 수정
* `BBButton` 내부에 있는 `StacView`로 인해 Touch Event가 cancel 되는 현상을 수정했습니다.
* `SplashViewController`에 있는 디버깅용`print` 구문을 제거했습니다.

## 🛠️BBButton 내부 mainStackView에 isUserInteractionEnabled프로퍼티 false 적용
- `BBButton`내부에 있는 `mainStackView`에 `isUserInteractionEnabled` 프로퍼티를 false로 적용하여 `UIButton` Touch Event를 막지 않도록 수정하였습니다.


## ✅테스트 케이스

* `BBButton` 사용하는 코드에서 Touch Event가 Cancel 되지 않는지 확인해요

---

#### 🙏🏻아래와 같이 PR을 리뷰해주세요.
- PR 내용이 부족하다면 보충 요청해주세요.
- 코드 스타일이 팀의 규칙에 맞게 작성되었는지, 일관성을 유지하고 있는지 확인해주세요.
- 코드에 대한 문서화나 주석이 필요한 부분에 적절하게 작성되어 있는지 확인해주세요.
- 구현된 로직이 효율적이고 올바르게 작성되었는지, 아키텍처를 잘 준수하고 있는지 검토해주세요.
- 네이밍, 포매팅, 주석 등 코드의 일관성이 유지되고 있는지 확인해주세요.

Do-hyun-Kim committed:
fix: SplashViewController print 구문 제거
- BBButton StackView isUserInteractionEnable fasle 처리

akrudal commented on 14th-team5-iOS/App/Sources/Presentation/Splash/SplashViewController.swift:
```diff
@@ -93,17 +93,17 @@ public final class SplashViewController: BaseViewController<SplashReactor> {
     private func showNextPage(with member: MemberInfo?) {
         @Navigator var splashNavigator: SplashNavigatorProtocol
-        print("memberId: \(member)")
+        
```
이왕이면 이 코드 reactor로 옮겨주시면 어때영..?ㅎㅎㅎ ㅎ헤

Do-hyun-Kim replied:
아하 넵 splashNavigator Reactor로 옮겨놓을게요

akrudal commented on 14th-team5-iOS/Core/Sources/Bibbi/BBCommons/BBButton/BBButton.swift:
```diff
@@ -81,6 +81,7 @@ public class BBButton: UIButton {
             $0.spacing = 4
             $0.distribution = .fillProportionally
             $0.axis = .horizontal
+            $0.isUserInteractionEnabled = false
```
GOOD!

Do-hyun-Kim committed:
fix: SplashViewController 화면 전환 로직 제거
- SplashReactor SplashNavigator 추가 및 화면 전환 로직 추가

akrudal reviewed (dismissed):
GOOD
이거 develop / release 둘ㄷ ㅏ 머지해야해영~

Do-hyun-Kim committed:
fix: SplashReactor Main화면 가는 함수 추가
- Bundle+Ext, API 파일 AppKey 수정
- FamilyInfoUserDefaults  familyId, familyName  String 타입 어노테이션 추가

Do-hyun-Kim committed:
fix: Tuist CFBundleShortVersionString 1.2.4로 수정
- Marketing Version String 1.2.4로 수정

akrudal reviewed (approved):
그만훼~~
